lib/chef/provider/package/rubygems.rb

```diff
@@ -401,7 +401,7 @@ def initialize(new_resource, run_context=nil)
         end
 
         def is_omnibus?
-          if RbConfig::CONFIG['bindir'] =~ %r!/opt/(opscode|chef)/embedded/bin!
+          if RbConfig::CONFIG['bindir'] =~ %r!/opt/(opscode|chef|chefdk)/embedded/bin!
```

Comment:
I'm on a big trip right now, and won't be able to look into this for a while. Let me know how you'd like to proceed from here.